api/src/legacy_soap_api/grantors/services/get_application_list_expanded_response.py

```diff
@@ -0,0 +1,85 @@
+import logging
+from datetime import datetime
+from zoneinfo import ZoneInfo
+
+from sqlalchemy import select
+from sqlalchemy.orm import joinedload
+
+import src.adapters.db as db
+from src.constants.lookup_constants import ApplicationStatus
+from src.db.models.competition_models import Application, ApplicationSubmission, Competition
+from src.legacy_soap_api.grantors import schemas
+
+logger = logging.getLogger(__name__)
+
+GRANTS_APPLICATION_STATUSES = {
+    None: None,
+    ApplicationStatus.IN_PROGRESS: None,
+    ApplicationStatus.SUBMITTED: "Received",
+    ApplicationStatus.ACCEPTED: "Validated",
+}
+
+
+def transform_submission(submission: ApplicationSubmission) -> dict[str, str | datetime | None]:
+    application = submission.application
+    competition = application.competition
+    opportunity = competition.opportunity
+    assistance_listing = competition.opportunity_assistance_listing
+    application_list_obj = {
+        "FundingOpportunityNumber": opportunity.opportunity_number,
+        "CFDANumber": assistance_listing.assistance_listing_number if assistance_listing else None,
+        "GrantsGovTrackingNumber": f"GRANT{submission.legacy_tracking_number}",
+        "GrantsGovApplicationStatus": GRANTS_APPLICATION_STATUSES[application.application_status],
+        "SubmissionTitle": application.application_name,
+        "PackageID": competition.legacy_package_id,
+        "n2:ReceivedDateTime": (
+            application.submitted_at.astimezone(ZoneInfo("America/New_York"))
+            if application.submitted_at
+            else None
+        ),
```

Comment:
It's similar, but we have a `datetime_util.adjust_timezone` function that would help simplify this a bit / has been tested to make sure it actually converts right.